Networking/Sources/Networking/Services/Emoji/EmojiModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  EmojiModel.swift
+//  
+//
+//  Created by DariaMikots on 07.07.2022.
+//
+
+import Foundation
+
+public struct Emoji: Decodable, Hashable {
+    
+    var id = UUID()
```

Comment:
For what is `id` for? All emojis are unique, so no ID is required
Or am I missing something? :)